services/hosted_services.md

```diff
@@ -1,19 +1,91 @@
+
+The materials in these lists primarily link to library electronic content hosting on our vendor resourses but they do include links to materials in the Library Catalog, Figgy and Finding Aids. They also contain links to digitized videos stored in the [Video Reserves application](https://github.com/PrincetonUniversityLibrary/video_reserves). 
+
+### Product Owners
+Our Resource Sharing staff are the primary staff users of Ares. Refer any functional questions about the application to them. 
+
+### Web Interface
+Atlas hosts the web interface code in a [Github repot](https://github.com/AtlasSystems/hosting-ares-princeton). You need to request access to this via Atlas support. [Documentation](https://support.atlas-sys.com/hc/en-us/articles/4407504126611-Editing-Atlas-Hosted-Aeon-Web-Pages-in-GitHub) on the workflow to allow PUL users to submit updates is available. Students, Faculty, or University staff who support the management of course reading lists typically do not log into the Ares web interface but rather interact with it through the Canvas integration. Resource sharing handle the creation of accounts to use the web interface directly by request on for those auditing courses who do not posses a Princeton net ID. 
+
+### Integration with Canvas
+[Canvas](https://princeton.instructure.com/) is Princeton's Learning Management System (LMS). Reading lists prepared for courses each semester are delivered through an integration with Ares supported by the [Learning Technology Interoperability](https://support.atlas-sys.com/hc/en-us/articles/5659979374483-Integrating-Ares-and-Canvas-with-LTI-1-3) (LTI) standard. Each active course that has a reading list created in Ares displays that list by linking the course identifier in Canvas with the course identifier in Aeon. There is an example test course in the system under "Library Sandbox" that can be utilized for testing Ares features in canvas. Only users who have successfully authenticated in canvas are able to see Ares contact. Ares content is available under the "Reserves" link in the individual course menu options. Ask the resource sharing staff about utilizing the [test reading list](https://princeton.instructure.com/courses/254/external_tools/399) if needed. 
+
+## Add-ons
+Aeon has an add-on architecture implemented in [LUA](https://www.lua.org/). Add-ons can be added at the individual client level or on the server. Client add-ons are incorporated into our Aeon client install in our staff workstation image and replicated on every client instance. Server add-ons are installed in a single place within the ILLiad server application. See available [add-ons](https://atlas-sys.atlassian.net/wiki/spaces/ILLiadAddons/pages/3149456/Ares+Addon+Directory).
```

Comment:
Does Aeon software also underpin the ILLiad application?

Reply:
It has no relation to ILLiad. They are "cousins" if you will. Both are products are atlas systems.

Reply:
Now I'm really confused. The text here says: "Server add-ons are installed in a single place within the ILLiad server application." If Aeon isn't related to ILLiad, why do we install add-ons for Aeon in an ILLiad server?

Reply:
This is copy and paste from the ILLiad page. I changed the name of the application to match the section. All three of the Atlas products have an identical add-on feature.